velox/functions/sparksql/Hash.cpp

```diff
@@ -363,4 +409,14 @@ std::shared_ptr<exec::VectorFunction> makeXxHash64(
   return kXxHash64Function;
 }
 
+std::shared_ptr<exec::VectorFunction> makeXxHash64WithSeed(
+    const std::string& name,
+    const std::vector<exec::VectorFunctionArg>& inputArgs,
+    const core::QueryConfig& /*config*/) {
+  const auto& constantSeed = inputArgs[0].constantValue;
+  VELOX_USER_CHECK_NOT_NULL(constantSeed, "Hash seed cannot be null");
```

Comment:
This check ensures that hash seed is constant, but it doesn't prevent the constant seed from being null. You need another check 

constantSeed->as<ConstantVector<int64_t>>()->isNullAt(0)